### DISCUSSION TITLE
continue use GitLens without Sign in to GitLens+

### DISCUSSION THREAD

**Discussion by Damian-Dev-Oracle** [gitkraken/vscode-gitlens#1941] [Feedback]
Hi Guys
I have a problem and i am confused how to continue use GitLens without Sign in to GitLens+
Options and features not appear anymore
![image](https://user-images.githubusercontent.com/58815439/161269266-35018545-f207-46b3-9571-5525e7d2d3b8.png)

any idea?

**Comment by Pirulax** [comment]
Yeah.. Same.. It's annoying.

**Comment by Pirulax** [comment]
Hey, update...
Seems like all the GitLens stuff has been moved to the source control tab:
<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/12902714/162622492-0f821a2c-849b-49c1-80d8-ac7933504c41.png)
</details>
Sorry for posting it kinda late, cause I did figure it out on the same day..

**Comment by Damian-Dev-Oracle** [comment]
no, I dont think so. It is default Git stuff for VS Code.
Before lovely "Trail" and on Free option whenever you opened file from repository then it automaticaly recognise it you can tracking changes. now nothing like that work. Hope it is only bug between paid and non-paid option. If no then unpaid option will become useless....

**Comment by eamodio** [comment]
GitLens, by default, doesn't add any "views" to the GitLens side bar and instead adds them to the Source Control side bar. The GitLens side bar is just the home of the new GitLens Home view, which provides access to GitLens+ features (and eventual teams & collaboration features). GitLens 12.0.6, should hopefully make this more clear with this new section on the Home view:

![image](https://user-images.githubusercontent.com/641685/163105479-24c3d03a-98e8-438a-9a6f-3877e9e6eb1c.png)

An account is only required to access the two new GitLens+ features (Visual File History and Worktrees). If you are having any other issues accessing any GitLens features, please open an issue with the details.

Hope that helps!

**Reply by Damian-Dev-Oracle** [reply]
I mean when I opened file (using VS Code) from repository usign windows eplorer then buttons on screen below loaded automatically and i was able to check history going onwards and backwords. Now i am not able to do it. To do it i have to go to Git tab, find branches and open file. I do not know if it is bug or something changed 
![image](https://user-images.githubusercontent.com/58815439/163361555-1bea6c29-b027-46bd-9168-0ae108f944d6.png)

**Reply by eamodio** [reply]
That sounds like a bug or a common cause is a folder casing issue. Could you please open an issue with more details on what you are seeing?

**Reply by Damian-Dev-Oracle** [reply]
not sure what you would like to see but in another words buttons above appear properly only when I open file from Git side tab from lists of your commits etc.
When you open file from Windows explorer (where git file are located on drive) then buttons not appear - I am not able to follow changes.
Those buttons appeared before - i am not able to describe it with more details.
No idea how to fix it
